### PR TITLE
feat: card title

### PR DESCRIPTION
**Pull request by jaulz**
### Motivation

This component implements `Card.Title` as described here:
https://material.io/design/components/cards.html#specs

### Test plan

From the example app:
image

image

image

### CODE REVIEW

File: src/components/Card/CardTitle.js [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+    flexDirection: 'row',
+    alignItems: 'center',
+    justifyContent: 'space-between',
+    paddingLeft: 16,
+  },
+
+  avatar: {
+    justifyContent: 'center',
+    marginRight: 16,
+    height: 40,
```
Comment (Trancever) [inline src/components/Card/CardTitle.js]: Shouldn't we also add `width: 40` here?

File: src/components/Card/CardContent.js [hunk 2]
```diff
@@ -57,23 +58,27 @@ class CardContent extends React.Component<Props> {
...
+      if (prev === cover || prev === title) {
         contentStyle = styles.only;
       } else {
         contentStyle = styles.last;
       }
-    } else if (prev === cover) {
+    } else if (prev === cover || prev === title) {
       contentStyle = styles.first;
-    } else if (next === cover) {
+    } else if (next === cover || prev === title) {
```
Comment (Trancever) [inline src/components/Card/CardContent.js]: Is this correct?

**Comment by Trancever** [comment]
Btw, This card with Title under Cover doesn't look good on screenshot, there is not enough space between them, what do you think @jaulz ?

File: src/components/Card/CardContent.js (hunk 2 above)
Comment (jaulz) [reply src/components/Card/CardContent.js]: No, it's fixed now, thanks 😊

File: src/components/Card/CardTitle.js (hunk 1 above)
Comment (jaulz) [reply src/components/Card/CardTitle.js]: Yep, added it.

**Comment by jaulz** [comment]
> Btw, This card with Title under Cover doesn't look good on screenshot, there is not enough space between them, what do you think @jaulz ?

Just updated the screenshots after the new commits 😊 You are right, some space was missing.

**Comment by Trancever** [comment]
Thanks for great work!

**Pull request merged**